### DISCUSSION TITLE
When did I create my github account? Where can I learn this?

### DISCUSSION THREAD

**Discussion by Err0r-IMA** [community/community#40970] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hello, When did I create my github account? Where can I learn this?

**Comment by radar-blip** [comment, marked as answer]
Click on the earliest date (below contribution graph)  

![joinedate](https://user-images.githubusercontent.com/39196767/206055483-7fa9ebc0-76da-4953-8ef1-6aafa273daad.png)


then keep clicking on Show more activity until you get where it says you joined.

![joined](https://user-images.githubusercontent.com/39196767/206055120-1852cf49-3d78-4244-8989-612c1d589e02.png)

**Reply by dov** [reply]
This is not necessarily correct if you pushed a repo with dates earlier than your registration date. You will then see these commits as activities in your account. E.g. I joined in 2010, but I have activities from as early as 2006.

**Comment by Omerasif3232** [comment]
> ### Select Topic Area
> Product Feedback
> 
> ### Body
> Hello, When did I create my github account? Where can I learn this?

**Comment by DosX-dev** [comment]
![image](https://github.com/community/community/assets/117774904/be5ecaba-f2e2-4293-bc27-5f5b9816dd5b)

Yes it works. thanks!

**Comment by yashu1wwww** [comment]
here you can fetch

https://githubuserjoineddatefinder.netlify.app/

**Comment by DhavalGojiya** [comment]
> ### Select Topic Area
> Product Feedback
> 
> ### Body
> Hello, When did I create my github account? Where can I learn this?

Visit: `https://api.github.com/users/<YOUR_GITHUB_USERNAME>`

Example:
`https://api.github.com/users/alice99`

**Reply by elcnqlyv** [reply]
That was the most helpful one. Thanks!

**Comment by YaroslavPodorvanov** [comment]
I made a small tool to make this easier:

**Git Profile Age** — https://doutivity.github.io/git-profile-age/

Just paste a GitHub profile URL and it automatically finds the account creation date using GitHub's public API. It also supports other Git hosting services such as GitLab, Bitbucket, Codeberg, Gitea and more.

No registration, API keys, analytics, or tracking. Everything runs directly in the browser.

Hopefully this makes it a bit easier than digging through the activity history or calling the API manually.
